packages/twenty-front/src/modules/ui/layout/selectable-list/hooks/useOnSelectableListEnter.ts

```diff
@@ -0,0 +1,41 @@
+import { useSelectableList } from '@/ui/layout/selectable-list/hooks/useSelectableList';
+import { useScopedHotkeys } from '@/ui/utilities/hotkey/hooks/useScopedHotkeys';
+import { getSnapshotValue } from '@/ui/utilities/recoil-scope/utils/getSnapshotValue';
+import { isNonEmptyString } from '@sniptt/guards';
+import { useRecoilCallback } from 'recoil';
+import { Key } from 'ts-key-enum';
+
+export const useOnSelectableListEnter = ({
```

Comment:
The naming is misleading here, it should be useListenToEnterHotkeyOnListItem something along this idea.